app/jobs/appointment_reminders/send_reminder_job.rb

```diff
@@ -0,0 +1,73 @@
+class AppointmentReminders::SendReminderJob
+  include Rails.application.routes.url_helpers
+  include Sidekiq::Worker
+
+  sidekiq_options queue: :high
+
+  # i think this should be based on country instead
+  DEFAULT_LOCALE = :en
```

Comment:
I may be wrong to think this should be based on country. I think defaulting to English is in line with how the rest of the app works. My guess is that most patients have an address, so their languages will be properly set.